src/Components/Navigation/Navigation.js

```diff
@@ -1,9 +1,236 @@
+            <div className="topMax">
+              <div className="upNavigation">
+                <div className="UpNavHost">
+                  <span>호스트 지원</span>
+                </div>
+                <div className="upNavContents">
+                  <Link to="/SignUP">
+                    <span>회원가입</span>
+                  </Link>
+                  <Link to="/SignIn">
+                    <span>로그인</span>
+                  </Link>
+                  <span>자주 묻는 질문</span>
+                  <span>공지사항</span>
+                </div>
+              </div>
+            </div>
+            <div className="midMax">
+              <div className="middleNavigation">
+                <CategoryImoBox>
+                  <img
+                    className="middleNavImo"
+                    src="/images/hamburger.png"
+                    alt="burger"
+                  />
+                  <div className="explainImo">카테고리</div>
+                </CategoryImoBox>
+
+                <span className="mainMiddleLogo">
+                  <img
+                    className="mainLogo"
+                    src="/images/logo.png"
+                    alt="mainLogo"
+                  />
+                </span>
+                <SearchBoxInNav>
+                  <input
+                    className="mainSearch"
+                    type="text"
+                    placeholder="검색어를 입력해 주세요."
+                  />
+                  <img
+                    className="loupeImo"
+                    src="/images/loupe.png"
+                    alt="loupe"
+                  />
+                </SearchBoxInNav>
+                <CategoryImoBox>
+                  <img
+                    className="middleNavImo"
+                    src="/images/book.png"
+                    alt="book"
+                  />
+                  <div className="explainImo">피드</div>
+                </CategoryImoBox>
+                <CategoryImoBox>
+                  <img
+                    className="middleNavImo"
+                    src="/images/bookmark.png"
+                    alt="bookmark"
+                  />
+                  <div className="explainImo">저장</div>
+                </CategoryImoBox>
+                <CategoryImoBox>
+                  <img
+                    className="middleNavImo"
+                    src="/images/user.png"
+                    alt="user"
+                  />
+                  <div className="explainImo">마이</div>
+                </CategoryImoBox>
+              </div>
+            </div>
+            <BottomMax>
+              <div className="bottomCategory">
+                <div>추천</div>
+                <div onClick={this.toggleSubNav}>일상</div>
+                <div>국내여행</div>
+                <div>베스트</div>
+                <div>기획전</div>
+              </div>
+            </BottomMax>
+          </div>
+        </FullNavBar>
+        <div>{this.state.subNavigationToggle ? <EventNavigation /> : ""}</div>
```

Comment:
```suggestion
        {this.state.subNavigationToggle && <EventNavigation />}
```
조건부 렌더링 (`&&` 연산자)를 활용하여 아래와 같이 바꿔볼 수 있습니다. 적용해보시기 바랍니다!

- 참고 : 공식문서 조건부 렌더링([https://ko.reactjs.org/docs/conditional-rendering.html](https://ko.reactjs.org/docs/conditional-rendering.html))

```jsx
true ? foo : null => true && foo // true인 경우에 foo 요소를 렌더
```

Reply:
수정하였습니다! 저런방법도 있었군요!! 감사합니다

Reply:
수정하였습니다!